cmd/kube-scheduler/app/options/options.go

```diff
@@ -158,7 +158,7 @@ func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 	o.Authorization.AddFlags(nfs.FlagSet("authorization"))
 	o.Deprecated.AddFlags(nfs.FlagSet("deprecated"), &o.ComponentConfig)
 
-	leaderelectionconfig.BindFlags(&o.ComponentConfig.LeaderElection, nfs.FlagSet("leader election"))
+	options.BindLeaderElectionFlags(&o.ComponentConfig.LeaderElection, nfs.FlagSet("leader election"))
```

Comment:
any reason this diverges from the common pattern of a `AddFlags` func?

Reply:
Looks like it wasn't placed in its own package

Reply:
The original `BindFlags` func in //pkg/client/leaderelectionconfig was moved to k8s.io/component-base/config/options/leaderelectionconfig.go due to the plan of making CCM easy to consume(kubernetes/cloud-provider#29). I thought `BindLeaderElectionFlags` as the name might be less confusing. If that is a common pattern, I will change it back. Is that expected?

Reply:
Oh, most options have a wrapper around the versioned config struct with an AddFlags() method bound to the type.

Given this is already in an options struct, a simple function seems ok.